### DISCUSSION TITLE
Way to identify Meross device after adding

### DISCUSSION THREAD

**Discussion by deeps2k** [krahabb/meross_lan#38] [General]
Have couple of Meross smart plugs , I was able to add all of them using IP address but after adding I am having hard time to identify those devices other than turning on and physically checking which is so painful if they are all connected in different room and are remote, Would be a great idea if it can show the IP address or under Device info or provide a way to identify them 
<img width="227" alt="image" src="https://user-images.githubusercontent.com/29079816/125655696-5b1167f9-02be-4328-9f53-0cb57a4f9952.png">

**Comment by krahabb** [comment]
Hello @deeps2k ,
Yeah, I see the point but, there's no 'official' way in HA to expose the IP address even though I could use that when creating the device name during initial configuration. But IP(s) could be volatile and so I would have an hard time updating those info in the device definition every time (not impossible but as I said there's no obvious api to update this). Of course, defining the ip right in configuration will 'set' that in stone and so my argument would fail but...
Anyway I will take this into consideration: the UUID I expose is a better way to correctly identify the device but I understand that's very abstract from an user point of view.
At the time you can still access the ip info from the integration configuration UI: by opening the integration configuration panel you'll see the IP address if the entry was setup like that
